### PR TITLE
fix(subscription/notify_admin): prevent invalid addition of datetime …

### PR DESCRIPTION
**Pull request by velrest**
…and int

### CODE REVIEW

File: timed/subscription/notify_admin.py [hunk 1]
```diff
@@ -21,8 +21,10 @@ def prepare_and_send_email(project, order_duration):
         minutes=duration.minute,
         seconds=duration.second,
     )
-    estimated_time = 0 if project.estimated_time is None else project.estimated_time
-    hours_total = hours_added + estimated_time
+
+    hours_total = hours_added
+    if project.estimated_time is not None:
+        hours_total += project.estimated_time
```
Comment (winged) [inline timed/subscription/notify_admin.py:27]: Why not be fully explicit and do `hours_total += datetime.timedelta(hours=project.estimated_time)`?

Comment (velrest) [reply timed/subscription/notify_admin.py:27]: I just tested and this will also throw an error if `project.estimated_time` is `None`.

Comment (winged) [reply timed/subscription/notify_admin.py:27]: Yeah the `None` check is required in every case

Comment (velrest) [reply timed/subscription/notify_admin.py:27]: But isnt estimated_time anyway a timedelta if its set?

Comment (winged) [reply timed/subscription/notify_admin.py:27]: Hmm, I thought it was an `int`, but you're right. Should be `timedelta` or `None`

**Review by winged — APPROVED** [review]

**Pull request merged**